Source/GitReleaseManager/Configuration/Config.cs

```diff
@@ -19,8 +19,11 @@ public Config()
                                   FooterHeading = string.Empty,
                                   FooterContent = string.Empty,
                                   FooterIncludesMilestone = false,
-                                  MilestoneReplaceText = string.Empty
-                              };
+                                  MilestoneReplaceText = string.Empty,
+                                  IncludeShaSection = false,
+                                  ShaSectionHeading = "__SHA256 Hashes of the release artifacts__",
+                                  ShaSectionLineFormat = "`{1}\t{0}`"

```

Comment:
Should this be in a list?
IE:
```suggestion
                                  ShaSectionLineFormat = "- `{1}\t{0}`"
```

Reply:
Yes, I think you are right.  I will fix this up.